config/htcondor-ce.spec

```diff
@@ -418,16 +445,21 @@ fi
 %{_datadir}/condor-ce/config.d/02-ce-bosco-defaults.conf
 
 %files client
+%config(noreplace) %{_sysconfdir}/condor-ce/condor_mapfile.osg
+%config(noreplace) %{_sysconfdir}/condor-ce/config.d/01-osg.conf
```

Comment:
Why is this mapfile and config going in the client?  Isn't it needed in the actual CE?

Reply:
If we can choose an arbitrary suffix other than `@opensciencegrid.org`, I think we could revert this change and have a single `condor_mapfile`. That being said, we should probably ship a `condor_config.template` and provide a config(noreplace) `condor_mapfile` if we really want to make this non-OSG-friendly.